x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/HttpClient.java

```diff
@@ -135,7 +159,7 @@ public void cancelled() {
     }
 
     private void respondUsingUtilityThread(HttpResponse response, HttpRequest request, ActionListener<HttpResult> listener) {
```

Comment:
This method name should be updated to reflect the fact that it no longer uses the utility thread pool.